src/config/vars.rs

```diff
@@ -45,7 +69,8 @@ mod test {
         assert_eq!("$ x", interpolate("$ x", &vars).0);
         assert_eq!("$FOO", interpolate("$$FOO", &vars).0);
         assert_eq!("", interpolate("$NOT_FOO", &vars).0);
-        assert_eq!("-FOO", interpolate("$NOT-FOO", &vars).0);
+        assert_eq!("", interpolate("$FOO-BAR", &vars).0);
```

Comment:
This changed since `-` _is_ an allowed character in variable names. I updated it to make it clearer that it doesn't just pull $FOO and leave `-BAR`.